min-turnup/phase2/ansible/playbooks/roles/master/templates/kube-apiserver.json.j2

```diff
@@ -0,0 +1,102 @@
+{
+  "apiVersion": "v1",
+  "kind": "Pod",
+  "metadata": {
+    "name": "kube-apiserver",
+    "namespace": "kube-system",
+    "labels": {
+      "tier": "control-plane",
+      "component": "kube-apiserver"
+    }
+  },
+  "spec": {
+    "hostNetwork": true,
+    "containers": [
+      {
+        "name": "kube-apiserver",
+        "image": "{{ phase2['docker_registry'] }}/hyperkube-amd64:{{ phase2['kubernetes_version'] }}",
+        "resources": {
+          "requests": {
+            "cpu": "250m"
+          }
+        },
+        "command": [
+             "/hyperkube",
+              "apiserver",
+              "--address=127.0.0.1",
+              "--etcd-servers=http://127.0.0.1:2379",
+              "--cloud-provider={{ phase1['cloud_provider'] }}",
+{% if phase1['cloud_provider'] == "azure" %}
+              "--cloud-config=/etc/kubernetes/azure.json",
```

Comment:
For comparison, this could be done with jsonnet like this:
```lua
local build_params(arr) =
  std.flattenArrays(std.filter(function(a) a != null, arr));

build_params([
  [
    "--this",
    "--that",
  ],
  if "gce" == "gce" then
    ["--life"],
  if "azure" == "gce" then
    ["--death"]
  else
    ["--parties"],
])
```
produces:
```json
[
   "--this",
   "--that",
   "--life",
   "--parties"
]
```